host-scripts/setup-node.sh

```diff
@@ -245,21 +78,35 @@ if ! id msk_jenkins > /dev/null 2>&1; then
   chmod u+rwX,go-rwX -R /home/msk_jenkins/.ssh
 fi
 
-if [ ! -f /root/.ssh/authorized_keys -o -z "`grep root@mskbuildhost /root/.ssh/authorized_keys`" ]; then
-  echo "Add ssh key for root..."
-  mkdir -p /root/.ssh
-  touch /root/.ssh/authorized_keys
-  echo ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDhl3/xPGCNgzSr20yF9D8s73b/ntH0w739whKCr4Gr47hVVuflTzVSSckbG7RHbCvubg272Xth3okMCJA1T8OAabGDiGWBg2iqHuH9WtTboZEaEHugcnMyGPRfVi+2XD5+WEKEhmcz+zXIu4MqqQkDrzjxtromxW1LlJFjD4bhwM7fPw14is9797IOPLxd/+nN9MQkj+91Idabf5byJeoS5instk/9axkHXXNariyOOHv7Yl5jx/9ceT3BZHPXu22VpgmicoArK6dZZx+VOJDYpy+T7IEaxiPCrFcdahLTEW0CGffp/gqcpYN9Cy4RRZMhQJ77wYarPhYdrXvyug3p root@mskbuildhost >> /root/.ssh/authorized_keys
-  echo ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAhUo63oSdWugxrG4AEJSZZfKydEDtAjMR2Ncq9v+mgdxuvSFZgPEDKio4VmX78aVG9oPLG5TeqxVv+2L/1F+vBNkow7dx6vHpuhawPuDo2wkupjjvoeTaaxKebC8xroGODl8BPdnJTELC9dOEdD+OPOzxK/0dtWB8iMvyL0Sau7aLmXiPJOPTCLwlFVMZl/uohJTy31i3HdYY7v79KHz6SOOWlTghIIodNkFR7r9oQwM7fNLnwoRLUWM1CsDb5umty5m2VdbZ25yKzqpE8cJp2eIQT5Y3dbM3st1xzK6IEsTN9VOkAvUSG9f8bqOJ8Qg9xvS+pl9babkayW6jwtQJ faimond@xfelbackup1 >> /root/.ssh/authorized_keys
-fi
```

Comment:
I think we need at least the first key so we can login as root to that VM.

Reply:
I thought that file was completely overwritten by puppet or was that for another host?

Reply:
That was on the head node, not inside the VM! Our VMs are not managed by Puppet.